packages/playwright/src/generators/configuration/files/__directory__/example.spec.ts.template

```diff
@@ -4,5 +4,5 @@ test('has title', async ({ page }) => {
   await page.goto('/');
 
   // Expect a title "to contain" a substring.
-  await expect(page).toHaveTitle(/Welcome/);
+  await expect(page).toHaveTitle(/<%= project %>/);
```

Comment:
did we confirm this on other projects as well?
I know it was working for next and angular before.

Reply:
i think it works for next because next title is `<title>Welcome to <%= name %>!</title>`: https://github.com/nrwl/nx/blob/master/packages/next/src/generators/application/files/pages/_app.tsx__tmpl__

but i don't think it would work for angular since its title is `<title><%= appName %></title>`:
https://github.com/nrwl/nx/blob/master/packages/angular/src/generators/application/files/base/src/index.html__tpl__

for react, title is "<NxWelcome title="<%= projectName %>"/>"
https://github.com/nrwl/nx/blob/master/packages/react/src/generators/host/files/common/src/app/__fileName__.tsx__tmpl__#L23

so i think change to to <%= project %> would make it work for angular and next

Reply:
actually i changed the test to query on h1 instead of title